pnpm-workspace.yaml

```diff
@@ -1,3 +1,3 @@
 packages:
-    - 'examples'
```

Comment:
but there's still a package.json in this folder right? should it be deleted too?